### DISCUSSION TITLE
Farming in different networks

### DISCUSSION THREAD

**Discussion by JS220313** [Chia-Network/chia-blockchain#9428] [Q&A]
Hi, 

I'm a begginer with Chia, and i'm wondering if someone can help me with my doubt, i have 2 machines in different places with different network ISP, so i would like to know if there is a way to both machines farm for one pool contract?, as example, if i have 20 plots in machine A connected to Network A, and 20 plots on machine B connected to network B, the total i farm in my pool will be 40 plots?

Hope i explained it well, 

Thanks !

**Comment by max918** [comment]
Yes, if all your plots on all machines was created by the same pool contract address.
https://github.com/Chia-Network/chia-blockchain/wiki/CLI-Commands-Reference#create


If you have more than 1 machine, chia suggest only run single main machine + N Harvester
https://github.com/Chia-Network/chia-blockchain/wiki/Farming-on-many-machines
